pikitlib/pikitlib/mecanumdrive.py

```diff
@@ -0,0 +1,77 @@
+from .speedcontroller import SpeedController
+from .robotdrivebase import RobotDriveBase
+
+import math
+
+class MecanumDrive(RobotDriveBase):
+
+    def __init__(self, frontleft: SpeedController, frontright: SpeedController, backleft: SpeedController, backright: SpeedController):
+        self.frontleft = frontleft
+        self.frontright = frontright
+        self.rearleft = rearleft
+        self.rearright = rearright
+        self.rightSideInvertMultiplier = -1.0
+        RobotDriveBase.__init__(self)
```

Comment:
I believe it is a better practice to call the base initializer *first thing* in the derived initializer.   This way, you know that you can use any of the base class methods safely.   (because you know that it has already been initialized).